src/python/WMComponent/DBS3Buffer/DBSBufferFile.py

```diff
@@ -387,8 +387,17 @@ def setProcessingVer(self, ver):
 
         Set the era
         """
-
-        self['processingVer'] = ver
+        procVer = 0 # type declaration
```

Comment:
this code change is likely not necessary (with the database schema change).

Reply:
I explicitly added type declaration not only for clarity but also to declare default value. It is bad practice to define new variable within if statements and my intention was to have proper definition. Please let me know explicitly if you disagree and I'll remove it then.

Reply:
I am in favor of validating the data type as it "enters" the system. Given the python duck typing nature, we would have to implement such code ALL over WMCore if we want to ensure data type, which in many cases will only unnecessarily increase the complexity of the system.

In this case, we already have two places enforcing the data type:
* at the spec level (WMSpec/StdBase)
* and at the database schema one

so I'd rather trust that those are enough for this check.

Reply:
ok, done